client/src/main/java/com/kritsit/casetracker/client/validation/DoubleValidator.java

```diff
@@ -7,7 +7,7 @@ public boolean validate(Object obj) {
         }
         String s = obj.toString();
         try {
-            double value = Double.parseDouble(s);
+            Double.parseDouble(s);
```

Comment:
"Dead store to local variable"

I was a bit worried about "Double.parseDouble(s);" getting optimized out without storing it but it passes the tests.